search-algos/binary_search/binary search.py

```diff
@@ -0,0 +1,38 @@
+print("Enter the number of elements you want to search from")
```

Comment:
This implementation is not very good,please take a look at binary_search.c code and run that code and see the difference.

Reply:
Ok will try to give more clean code.
Please review 0-1 knapsack also